test/integration/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const bodyParser = require('body-parser');
+const express = require('express');
+const http = require('http');
+const logging = require('@google-cloud/logging')();
+const monitoring = require('@google-cloud/monitoring').v3();
+
+// Application implementing the integration test spec at
+// https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/integration_tests/README.md
+const app = express();
+
+app.use(bodyParser.json());
+app.get('/', (req, res) => {
+  res.status(200).send('Hello World!');
+});
+
+app.post('/logging_standard', (req, res) => {
```

Comment:
Is it required to implement a set of expected paths? I don't think this is something we would expect to implement in the future for Node.js.